### ISSUE TITLE
[Pet] Hearty

### ISSUE THREAD

**Issue by Ragelezz** [Evolution-WoW/Bug-Report#737]
https://www.wowhead.com/item=163218/hearthy not implemented in Legion.
npc id 117343

**Comment by skeezerbean** [comment]
Correct, it was implemented in SPP custom by Duff, and this is what happens when you apply half-assed fixes from other projects with less IQ than a box of rocks, especially if you're a known scammer selling fake source updates. The moment you learn to actually do real things, then these problems won't be popping up and slamming you in the pooper. For more information, learn to code, learn to pull fixes from other projects without being a complete screwup, and go back in time to give your dad a condom. That should probably be enough to fix this issue.


![image](https://user-images.githubusercontent.com/19826665/160663009-804686de-ad30-41d6-a9b5-6c1ac9eb2e04.png)
![image](https://user-images.githubusercontent.com/19826665/160663040-1d0b2d83-7cd7-48f2-85ff-8c8712afc631.png)
![image](https://user-images.githubusercontent.com/19826665/160663089-88dbc8e8-8859-430c-b9d0-8d95c8125777.png)

**Comment by phantom1004** [comment]
https://youtu.be/N2wzD5SCWtc
